### PR TITLE
Added `mask-generation` and `zero-shot-object-detection`

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @osanseviero

### CODE REVIEW

File: packages/tasks/src/pipelines.ts
Comment (mertssmnoglu) [inline packages/tasks/src/pipelines.ts:1]: ```name: "Mask Generation"``` :white_check_mark: It follows the rules
```name:     "Mask Generation"``` :x: Doesn't follow the rules

Can be easily fixed with `pnpm -r format`

**Review by osanseviero — APPROVED** [review]
Nice! This looks complete! :fire: Would you mind posting a screenshot to see how the icons would look like?

cc @gary149 for the icons

**Comment by merveenoyan** [comment]
One is mask generation and the other one is zero shot object detection @gary149 @osanseviero 
Screenshot 2023-11-22 at 14 25 57 1
Screenshot 2023-11-22 at 14 26 31

**Comment by gary149** [comment]
Ok I updated the icons.

image

**Comment by merveenoyan** [comment]
Should I go ahead and merge?

**Pull request merged**